server/lib/scanners/jellyfin/index.ts

```diff
@@ -96,7 +96,7 @@ class JellyfinScanner {
           (MediaStream) => MediaStream.Type === 'Video'
         ).some((MediaStream) => {
           return (MediaStream.Width ?? 0) <= 2000;
-        });
+        }) || MediaSource.Container === 'strm';
```

Comment:
By doing this, all STRM media will be considered non-4k media. Right now I don't know what method to use, but you have to find a way to retrieve the media definition.

Reply:
---hm, I also have no ide.